src/chess/chess-arbiter.js

```diff
@@ -9,17 +9,18 @@ AFRAME.registerSystem("chess-arbiter", {
     this.chessGame = this.sceneEl.querySelector("a-entity[chess-game]");
     this.sceneEl.addEventListener("chess-command", ev => {
       const command = ev.detail[0];
-      this.handleChatCommand(command);
+      const param = ev.detail[1];
+      this.handleChatCommand(command, param);
     });
     this.resetGame();
     GameNetwork.setupNetwork(this.sceneEl);
   },
 
   tick() {
     if (this.state.imPlaying) {
-      const pieces = this.sceneEl.querySelector("a-entity[chess-set]").children;
+      const pieces = Array.prototype.slice.call(this.sceneEl.querySelector("a-entity[chess-set]").children).map(x => x.firstChild)
```

Comment:
```suggestion
      const pieces = this.sceneEl.querySelectorAll("a-entity[chess-set] a-entity[chess-piece] a-entity.chess-piece");
```

careful with `firstChild`, it can return a text node. Safer to use `firstElementChild` or just tweak the selector